api/graphql/types/reservation_unit/serializers.py

```diff
@@ -118,6 +121,53 @@ def validate(self, data: dict[str, Any]) -> dict[str, Any]:
 
         return data
 
+    def validate_reservation_duration_fields(self, data: dict[str, Any]) -> None:
+        """
+        Validates:
+        - min_reservation_duration
+        - max_reservation_duration
+        - reservation_start_interval
+        """
+        min_duration: timedelta | None = data.get("min_reservation_duration")
```

Comment:
Use `self.get_or_default("min_reservation_duration", data)` (part of `NestingModelSerializer`), which will give you the current instance value if "min_reservation_duration" is not given in the update-mutation data. It'll also use the model field default value in create-mutation if the field is not required and not given there (though if the field has not default, this will be a [Sentinel](https://en.wikipedia.org/wiki/Sentinel_value) value "NotProvided", which is falsy)

Reply:
Changed those `get`s to `get_or_default`s